
FOB_controller:
    type: phri_fob/FOB_controller
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7

collision_config:
    lower_torque_thresholds_nominal: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0]
    upper_torque_thresholds_nominal: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0]
    lower_force_thresholds_nominal:   [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]
    upper_force_thresholds_nominal:   [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]


    
